Services/AccessControl/classes/class.ilAccess.php

```diff
@@ -899,4 +899,18 @@ public function filterUserIdsByPositionOfUser($user_id, $pos_perm, $ref_id, arra
 	public function filterUserIdsByRbacOrPositionOfCurrentUser($rbac_perm, $pos_perm, $ref_id, array $user_ids) {
 		return $this->ilOrgUnitPositionAccess->filterUserIdsByRbacOrPositionOfCurrentUser($rbac_perm, $pos_perm, $ref_id,  $user_ids);
 	}
+
+	/**
+	 * @inheritdoc
+	 */
+	public function hasCurrentUserAnyPositionAccess($ref_id) {
```

Comment:
@smeyer-ilias there are several new method in the position-related interfaces which we had to implement in ilAccess, too. OK for you?